packages/create-cloudflare/src/common.ts

```diff
@@ -205,7 +205,10 @@ export const chooseAccount = async (ctx: PagesGeneratorContext) => {
 
 export const printSummary = async (ctx: PagesGeneratorContext) => {
 	const nextSteps = [
-		[`Navigate to the new directory`, `cd ${ctx.project.name}`],
+		[
+			`Navigate to the new directory`,
+			`cd ${relative(ctx.originalCWD, ctx.project.path)}`,
```

Comment:
If the path contains spaces, this will log out an invalid command

Wherever we log/use stringified commands, we should use `shell-quote`